packages/st2/rpm/postinst_script.spec

```diff
@@ -1,4 +1,8 @@
 set -e
 
-# make sure that our socket generators run
+# Reload systemd to run generators for unit and socket files.
 systemctl daemon-reload >/dev/null 2>&1 || true
+
+# Enable services created by systemd generator
+systemctl enable st2api st2auth st2stream >/dev/null 2>&1  || true
+systemctl start st2api st2auth st2stream >/dev/null 2>&1  || true
```

Comment:
Looking at the test error for EL distros:

I think this is handled by the `%post` macro which uses the `%service_post` helper. That runs:
https://github.com/StackStorm/st2-packages/blob/23f100cbe709678cbdae81055b8c7dd244c2b883/rpmspec/helpers.spec#L86

What if you remove the output redirection in the `%service_post` macro helper? Then maybe we'll see if there's an error. And maybe remove it here too.

Reply:
There appears to be a worker timeout for gunicorn on EL7, but I don't think that's the cause of the services failing to start for the RPM based distros.  I still don't see any errors in the logs that indicate why the service is not enabled and started.